### PR TITLE
Switching previously bound functions to arrow functions in Edge.js

### PR DESCRIPTION
**Pull request by kenneth-miura**
Tested by selecting and unselecting Nodes in Courseography, verifying that it is able to render an edge in all 4 ways.

### CODE REVIEW

File: js/components/graph/Edge.js [hunk 1]
```diff
@@ -26,6 +25,7 @@ export default class Edge extends React.Component {
     }
   }
 
+
```
Comment (david-yz-liu) [inline js/components/graph/Edge.js:28]: Please delete this line.

File: js/components/graph/Edge.js [hunk 2]
```diff
@@ -25,7 +24,7 @@ export default class Edge extends React.Component {
       this.setState({ status: "active" });
     }
   }
-
+
```
Comment (david-yz-liu) [inline js/components/graph/Edge.js:27]: Please delete the wuitespace on this line.

Comment (kenneth-miura) [reply js/components/graph/Edge.js]: This comment is referring to line 27 right?

Comment (david-yz-liu) [reply js/components/graph/Edge.js]: Yes that's correct; GitHub shows the comment immediately below the line that it's referring to.

**Pull request merged**